resources/js/app.js

```diff
@@ -1,11 +1,12 @@
-import Vue from 'vue';
-import VueRouter from 'vue-router';
-import routes from './routes';
+require('./bootstrap');
+window.Vue = require('vue');
 
-Vue.use(VueRouter);
 Vue.prototype.$bus = new Vue();
```

Comment:
Do you think you could have implemented this app without using an event bus (i.e. just using component events)?

Reply:
There were a few options I had to resolve this but I went with the event bus because I originally thought the problem I was solving wasn't complex enough for Vuex and I didn't want to pass props and emit event across child components that wouldn't need that data.